#! /bin/sh /usr/share/dpatch/dpatch-run
## makefile_distclean.dpatch by Sheldon Hearn <sheldonh@starjuice.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add distclean target to top-level Makefile

@DPATCH@

--- fsvs-1.1.11.orig/Makefile
+++ fsvs-1.1.11/Makefile
@@ -16,3 +16,8 @@
 configure:	configure.in
 	@echo Generating configure.
 	autoconf
+
+distclean:
+	rm -f config.cache config.log config.status 2> /dev/null || true
+	rm -f src/Makefile src/tags tests/Makefile 2> /dev/null || true
+	rm -f src/config.h src/*.[os] src/.*.d src/fsvs 2> /dev/null || true
